control-station/src/App.tsx

```diff
@@ -1,7 +1,9 @@
 import { ControlPanel, Navbar } from "@/components";
 import usePodData from "./services/usePodData";
 import PodContext from "./services/PodContext";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { Dashboard } from "@/views";
+import Dynamics from "./components/Dynamics/Dynamics";
```

Comment:
For consistency, let's move the `Dynamics` page to `views` like we have with `Dashboard`